### ISSUE TITLE
Option 18 support

### ISSUE THREAD

**Issue by jpc2350** [HenriWahl/dhcpy6d#9]
Wondering whether option-18 would be supported in the near future? I'm looking to get it to support the requirement I have for Option-18.

**Comment by HenriWahl** [comment]
This is needed for DHCPv6 relaying, right? Relaying is planned but I cannot promise when.

**Comment by jpc2350** [comment]
Yeah it's the interface-id specifier for relaying. Ok, thanks for the info.

**Comment by HenriWahl** [comment]
What requirements do you have for option 18?

**Comment by jpc2350** [comment]
So we're looking to assign PD's to CPE's based on the interface_id that is passed from our L2 switches, relayed through our core to our DHCP servers, it's required for us to identify our customers to our multitude of platforms and assign them the correct PD based on their service.

**Comment by HenriWahl** [comment]
This would be the next feature which dhcpy6d should have - PD.

**Comment by tomaszmrugalski** [comment]
Have you looked at Kea (http://kea.isc.org or http://github.com/isc-projects/kea)? It does support PD, interface-id and relaying.

**Comment by jpc2350** [comment]
Yeah, we're looking at Kea as well. Unfortunately, at the moment, it doesn't support interface-id (option 18) either, but we are engaging with ISC regarding that as well.

**Comment by jpc2350** [comment]
Unless it now does, and their documentation is out of date?
